var/spack/repos/builtin/packages/libszip/package.py

```diff
@@ -36,11 +36,11 @@ class Libszip(AutotoolsPackage):
     homepage = "https://support.hdfgroup.org/doc_resource/SZIP/"
     url      = "https://support.hdfgroup.org/ftp/lib-external/szip/2.1.1/src/szip-2.1.1.tar.gz"
     list_url = "https://support.hdfgroup.org/ftp/lib-external/szip"
-    list_depth = 2
+    list_depth = 3
 
     provides('szip')
 
-    version('2.1.1', 'dd579cf0f26d44afd10a0ad7291fc282')
+    version('2.1.1', '5addbf2a5b1bf928b92c47286e921f72')
```

Comment:
Can you confirm with the `szip` developers that this checksum change is intended? Just trying to prevent one of these: https://github.com/caskroom/homebrew-cask/issues/33536

Reply:
I have compared contents of the old and the new archives. They are identical but the new one gets unpacked without strange warnings.